olp-cpp-sdk-authentication/include/olp/authentication/AuthenticationClient.h

```diff
@@ -148,6 +148,54 @@ class AUTHENTICATION_API AuthenticationClient {
     unsigned int expires_in{0};
   };
 
+  /**
+   * @brief The Apple sign-in properties structure.
+   */
+  struct AppleSignInProperties {
+    /**
+     * @brief The Apple id_token to be used as bearer in the authorization
+     * request header.
+     */
+    std::string access_token;
+
+    /**
+     * @brief OIDC name property for the user.
```

Comment:
Can we put it in the following way?
```suggestion
     * @brief Your OIDC name.
```